### PR TITLE
adjusted anchor tag to div and added css to change to pointer // darker text log out and admin name

### PR DESCRIPTION
**Pull request by frankmfernandez**
changed cursor on log out to hand

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -12115,4 +12115,14 @@ body.sidebar-toggled footer.sticky-footer {
 [data-tip]:hover:before,
 [data-tip]:hover:after {
 	display:block;
-}
\ No newline at end of file
+}
+
+#userDropdown{
+
+color:rgba(108, 122, 137, 1);
```
Comment (wilandrade) [inline src/styles/App.css]: Please add proper spacing. Missing a tab here?

File: src/components/Dashboard.jsx [hunk 2]
```diff
@@ -105,13 +105,13 @@ class Dashboard extends Component {
                     </li>
                     <div className="topbar-divider d-none d-sm-block"></div>
                     <li className="nav-item dropdown no-arrow">
-                      <span
+                      <div
```
Comment (wilandrade) [inline src/components/Dashboard.jsx]: Maybe revert this back to span

**Review by wilandrade — CHANGES_REQUESTED** [review]

File: src/styles/App.css (hunk 1 above)
Comment (frankmfernandez) [reply src/styles/App.css]: should be fixed now

File: src/components/Dashboard.jsx (hunk 2 above)
Comment (frankmfernandez) [reply src/components/Dashboard.jsx]: ok

**Review by wilandrade — APPROVED** [review]

**Pull request merged**